src/entities/Proposal/routes.ts

```diff
@@ -377,6 +390,13 @@ export async function createProposal(
   const start = Time.utc().set('seconds', 0)
   const end = data.finish_at
   const proposal_url = proposalUrl({ id })
+  let coAuthors: string[] | null = null
+
+  if (data.configuration && data.configuration.coAuthors) {
+    coAuthors = data.configuration.coAuthors
+    delete data.configuration.coAuthors
```

Comment:
Is it necessary to delete this from configuration?

Reply:
I don't see why it should be stored in configuration as well if we have a specific table for this. WDYT?